_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -246,7 +246,7 @@ release candidates.*
   using the transaction's txid, which is a hash over the legacy fields
   of the transaction.  The new fields used in segwit transactions are
   not included in the txid, which was necessary to eliminate third-party
-  and counterparty transaction malleability.  However, during his June 2018 review
+  and counterparty transaction malleability.  However, during his June 2016 review
```

Comment:
ACK. I should have noticed the date (Jun 28, 2016) of the blog post: https://petertodd.org/2016/segwit-consensus-critical-code-review. Thanks @petertodd.